source/_docs/troubleshooting_general.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: "General troubleshooting"
+description: "General troubleshooting information"
+---
+
+This page provides some information about more generic troubleshooting topics.
+
+## Home Assistant went into recovery mode
+
+### Symptom: Home Assistant is in recovery mode
+
+On top of the page you see a red banner. On the **Overview** page, you see a **Recovery mode** notification.
+
+![image](/images/docs/troubleshooting/recovery_mode_active.png)
+
+### Description
+
+When Home Assistant is in recovery mode, there was an issue with the configuration.
+
+Recovery mode loads a few minimal integrations and parts of the configuration that was used the last time when it did start. You can still see the user interface, the settings, and add-ons.
+
+### Resolution
+
+You need to identify the issue in the configuration files and fix it there. The issue could be caused by something as simple as an invalid YAML file.
+
+- If you are still logged in, you can [edit your configuration](/docs/configuration/#editing-configurationyaml).
+  - In the Home Assistant user interface, open the add-on you usually use and edit the configuration file.
```

Comment:
I'm not sure if this page is only intended for users running an installation that supports add-ons. Editing the configuration file in other installation types isn't done via an add-on.